File: app/src/main/java/com/mxdl/kotlin/bean/Person1.java
package com.mxdl.kotlin.bean;

/**
 * Description: <Person1><br>
 * Author:      mxdl<br>
 * Date:        2021/1/3<br>
 * Version:     V1.0.0<br>
 * Update:     <br>
 */
class Person1 {
    private String name;
    private int age;

    public Person1(String name, int age) {
        this.name = name;
        this.age = age;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public int getAge() {
        return age;
    }

    public void setAge(int age) {
        this.age = age;
    }
}

File: app/src/main/java/com/mxdl/kotlin/test/Person1.kt
package com.mxdl.kotlin.test

/**
 * Description: <Person><br>
 * Author:      mxdl<br>
 * Date:        2020/12/13<br>
 * Version:     V1.0.0<br>
 * Update:     <br>
 */
open class Person1 constructor(var name:String,var age:Int){
    fun eat(){
        println(name+" is eating.he is "+age+" years old")
    }
}

fun main(args: Array<String>) {
    var person = Person()
    person.name = "Jack"
    person.age = 18
    person.eat()
}